src/wavesurfer.js

```diff
@@ -270,6 +270,7 @@ export default class WaveSurfer extends util.Observer {
         forceDecode: false,
         height: 128,
         hideScrollbar: false,
+        hideCursor: false,
```

Comment:
also needs to be documented at top of this file.

and changelog entry is missing.